### ISSUE TITLE
Mint Shutting Down Jan 1, 2024

### ISSUE THREAD

**Issue by ovitch** [jprouty/mint-amazon-tagger#160]
Wanted to make sure everyone saw the announcement. Curious where you guys will move your data? 

Heard some people are considering Quicken but just starting my search... Tiller Financial seems too basic in my book and not a big fan of putting my data on G.Sheet.

**Comment by smenzer** [comment]
I’ve started a trial with Monarch, seems like a pretty solid alternative so
far.

**Comment by jprouty** [comment]
I have tried just about all of the available alternatives and I've landed on Monarch money as my "favorite". There are shortcomings of course and things are in different places but I like the budget views over different time periods. I like it smoothed out when doing spend analysis, so I use the annual view.

**Comment by jprouty** [comment]
I've been in touch with their support team already about some various asks and if I can access an API (answer was no).

They did however point out their [roadmap]https://portal.productboard.com/3qsdvcsy5aq69hhkycf4dtpi/tabs/1-ideas), which has an "Amazon connector" under the ideas section. I'd like to think this tool specifically gave them the idea 🤣👍.

https://github.com/jprouty/mint-amazon-tagger/issues/150 is the future of where I'd like to take this endeavor, which would be universally more useful (more merchants and more tools - and even financial institution websites!).

**Comment by ovitch** [comment]
@jprouty this is good news, at least it's on their radar. But I wonder if AMZN is really the limiting factor here. Never found details on API for regular customers on amazon docs.

**Comment by jprouty** [comment]
Right. I am of the mindset that Amazon is data liberation adverse. e.g. In 2015 Amazon stopped providing detailed order invoices via email and instead sending you links to come back to Amazon.com to view status/receipts.

**Comment by ovitch** [comment]
Good thread here to contrast both: https://www.reddit.com/r/copilotmoney/comments/17l10qp/has_anyone_here_used_monarch_money_and_chosen_one/

Main drawback is copilot is not a webapp (+ need to check how you can export your data for storage)

**Comment by ovitch** [comment]
Agreed. How do you guys plan to save your financial data? I liked that mint allowed you to export a csv for recordkeeping.

**Comment by jprouty** [comment]
Monarch Money has the ability to save a CSV of transactions in (at least) 2 places:
- Transaction page, you can download the current view (either all, or based on the current filter parameters)
- Account detail page, click the Edit button and select "Download transactions". This is per account. In this same place, you can download the per-account balance history (daily).

**Comment by ovitch** [comment]
Thank @jprouty I am going to experiment this week with both Monarch and Copilot. 

In the meantime, I'll run the tagger until the last minute!

**Comment by jprouty** [comment]
Let the good times roll 🎢 !

**Comment by cwmoriarty** [comment]
Looks like they've pushed this until late March. 

Hopefully Copilot, Monarch, et al will still offer discounts during that time.

**Comment by jprouty** [comment]
Oh thanks for the continued intel! For those looking to follow along, [this is the source I found](https://support.creditkarma.com/s/article/Intuit-Mint-and-Credit-Karma#Can-I-keep-using-Mint) for a March 23rd sunset date.
